lib/mocha.js

```diff
@@ -483,21 +512,32 @@ Mocha.prototype.run = function(fn) {
   if (options.globals) {
     runner.globals(options.globals);
   }
+  // Use only the first reporter for growl, since we don't want to
+  // send several notifications for the same test suite
   if (options.growl) {
-    this._growl(runner, reporter);
+    this._growl(runner, reporters[0]);
```

Comment:
Unrelated to the PR but, apparently, we pass a reporter to `this._growl` because we get `failures` and `duration` from the Reporter.

IMO that info belongs to the Runner. Do you agree @boneskull? Just from looking at the Runner you should be able to tell whether it's running or not, for how long it's been running, how many pass/fails so far, etc.